src/shared/route-urls.ts

```diff
@@ -79,7 +79,7 @@ export enum RouteUrls {
   SendOrdinalInscription = 'send/ordinal-inscription',
   SendOrdinalInscriptionChooseFee = 'choose-fee',
   SendOrdinalInscriptionReview = 'review',
-  SendOrdinalInscriptionSummary = 'send/ordinal-inscription/',
+  SendOrdinalInscriptionSummary = 'summary',
```

Comment:
Maybe we can just delete this as I can't find it used anywhere
```suggestion
```

Reply:
🪓